src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/JiraRteEditPageTest.kt

```diff
@@ -0,0 +1,174 @@
+import com.atlassian.performance.tools.jiraactions.api.memories.IssueKeyMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.User
+import com.atlassian.performance.tools.jiraactions.api.memories.UserMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveIssueKeyMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveIssueMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveJqlMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveProjectMemory
+import com.atlassian.performance.tools.jiraactions.api.w3c.DisabledW3cPerformanceTimeline
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import java.time.Clock
+import java.util.*
+
+/**
+ * Test Editing issue without SetupAction (i.e Rich Text editor enabled)
+ * Covers:
+ *      EditIssue
+ *      AddComment
+ */
+class JiraRteEditPageTest {
+
+    @Test
+    fun shouldRunScenarioWithoutErrors() {
+        val scenario = JiraEditScenario()
+        val metrics = mutableListOf<ActionMetric>()
+        val actionMeter = ActionMeter(
+            virtualUser = UUID.randomUUID(),
+            output = CollectionActionMetricOutput(metrics),
+            clock = Clock.systemUTC(),
+            w3cPerformanceTimeline = DisabledW3cPerformanceTimeline()
+        )
+        val user = User("admin", "admin")
+        val userMemory = object : UserMemory {
+            override fun recall(): User {
+                return user
+            }
+
+            override fun remember(memories: Collection<User>) {
+                throw Exception("not implemented")
+            }
+        }
+
+        JiraCoreFormula.Builder()
+            .build()
+            .provision()
+            .use { jira ->
+                DockerisedChrome().start().use { browser ->
+                    val webJira = WebJira(
+                        browser.driver,
+                        jira.getUri(),
+                        user.password
+                    )
+                    val logInAction = scenario.getLogInAction(
+                        webJira,
+                        actionMeter,
+                        userMemory
+                    )
+                    val actions = scenario.getActions(
+                        webJira,
+                        SeededRandom(123),
+                        actionMeter
+                    )
+
+                    logInAction.run()
+                    actions.forEach { action ->
+                        action.run()
+                    }
+                }
+            }
+
+        val results = metrics.map { metric ->
+            metric.result
+        }
+        Assertions.assertThat(results).containsOnly(ActionResult.OK)
+    }
+}
+
+class JiraEditScenario constructor() : Scenario {
+    private lateinit var issueKeyMemory: IssueKeyMemory
+
+    private constructor(issueKeyMemory: IssueKeyMemory?) : this() {
```

Comment:
We can avoid the entire `issueKeyMemory` injection. It's present in `JiraCoreScenario` for API consumers, which want to share memories across scenarios. We don't need that feature in this test.
Let's replace it with a simple local variable.